db/migrate/20160619064544_change_memory_reserve_data_type_in_vm.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeMemoryReserveDataTypeInVm < ActiveRecord::Migration[5.0]
+  def up
+    change_column :vms, :memory_reserve, :bigint
```

Comment:
@durandom Yes, megabytes is the default unfortunately.  Moving forward, I'd like to have all these changed into a common "base unit" so there's no more guessing at the type.  We already do this in many places where "bytes" is the bast unit, but columns like this one have not been changed yet.